TAGradingServer/account/index.php

```diff
@@ -405,56 +412,145 @@
+    document.getElementById('container').style.height = (window.innerHeight - 40) + 'px';
 
     var split = 0;
-    var width = window.innerWidth - 8 - 2 - 2;
-    split = <?php echo ($split == "" || $split < 50 || 75 < $split ? 65 : $split); ?> / 100.0;
+    var width = window.innerWidth - 7;
+    split = <?php echo ($split == "" || $split < 25 || 75 < $split ? 50 : $split); ?> / 100.0;
     if (document.getElementById('left') != null) {
         document.getElementById('left').style.width = (width * (split)) + 'px';
     }
     if (document.getElementById('right') != null) {
-        document.getElementById('right').style.width = ((width * (1.0 - split)) - 10) + 'px';
+        document.getElementById('right').style.width = ((width * (1.0 - split)) ) + 'px';
     }
     if (document.getElementById('panemover') != null) {
         document.getElementById('panemover').style.left = document.getElementById('pane').offsetLeft + 'px';
         document.getElementById('panemover').style.width = '10px';
     }
 
+    window.onload = updateDisplay();
+
     window.onresize = function(event) {
-        width = (window.innerWidth - 8 - 2 - 2);
+        document.getElementById('grade').style.left = parseFloat(window.innerWidth * document.getElementById('grade').offsetLeft/(width + 7)) + 'px';
+        document.getElementById('rubric').style.left = parseFloat(window.innerWidth * document.getElementById('rubric').offsetLeft/(width + 7)) + 'px';
+        document.getElementById('stats').style.left = parseFloat(window.innerWidth * document.getElementById('stats').offsetLeft/(width + 7)) + 'px';
+
+        width = window.innerWidth - 7;
         split = (parseInt(document.getElementById('left').style.width) / (parseInt(document.getElementById('left').style.width) + parseInt(document.getElementById('right').style.width))).toFixed(2);
 
-        document.getElementById('left').style.width = (width * (split)) + 'px';
-        document.getElementById('right').style.width = ((width * (1 - split)) - 10) + 'px';
+        document.getElementById('left').style.width = (width * split) + 'px';
+        document.getElementById('right').style.width = (width * (1 - split)) + 'px';
 
-        document.getElementById('prog').style.width = (document.documentElement.clientWidth - 157) + 'px';
+        // document.getElementById('prog').style.width = (document.documentElement.clientWidth - 157) + 'px';
         document.getElementById('container').style.width = window.innerWidth + 'px';
-        document.getElementById('container').style.height = (window.innerHeight - 20 - 40) + 'px';
+        document.getElementById('container').style.height = (window.innerHeight - 40) + 'px';
         document.getElementById('panemover').style.left = document.getElementById('pane').offsetLeft + 'px';
+        updateDisplay();
     };
 
-    function dragStart(e, left, right) {
+    window.onkeydown = function(e) {
```

Comment:
indent four spaces on this function and the next one instead of the random switch to 2 spaces